src/coefficients/stdfunction_coefficient.cpp

```diff
@@ -44,18 +46,20 @@ void MakeEssList(mfem::Mesh &m, mfem::VectorCoefficient &c, mfem::Array<int> & e
   ess_vdof_list.SetSize(0);
 
   for (int v = 0; v < fes.GetNV(); v++)
```

Comment:
Is there an option to enforce { on all if's and for's. Because this is just asking for very hard to find mistakes.

Reply:
I think clang-format is supposed to only do whitespace changes, so inserting characters to conform to the guidelines isn't an option. clang-tidy is capable of more complicated changes, but is more work to set up and maintain.

I prefer for loops with braces as well, so I'll go ahead and manually insert them in this case.